<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="Title">
               <property name="text">
                <string>Enter IPv4 adress 1:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="firstBit"/>
               </item>
               <item>
                <widget class="QLabel" name="dot1">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="secondBit"/>
               </item>
               <item>
                <widget class="QLabel" name="dot2">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="thirdBit"/>
               </item>
               <item>
                <widget class="QLabel" name="dot3">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="fourthBit"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="checkOne">
               <property name="text">
                <string>Check</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="3" column="1">
                <widget class="QLabel" name="classDcheck">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="classEcheck">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="cC">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="cB">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="classCcheck">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="cE">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="cA">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="classBcheck">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="classAcheck">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="cD">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="Title_2">
               <property name="text">
                <string>Enter IPv4 adress 2:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="firstBit_2"/>
               </item>
               <item>
                <widget class="QLabel" name="dot1_2">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="secondBit_2"/>
               </item>
               <item>
                <widget class="QLabel" name="dot2_2">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="thirdBit_2"/>
               </item>
               <item>
                <widget class="QLabel" name="dot3_2">
                 <property name="font">
                  <font>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="fourthBit_2"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="checkTwo">
               <property name="text">
                <string>Check</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="3" column="1">
                <widget class="QLabel" name="classDcheck_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="classEcheck_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="cC_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="cB_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="classCcheck_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="cE_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="cA_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="classBcheck_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="classAcheck_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="cD_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
